### ISSUE TITLE
README needs enhancments

### ISSUE THREAD

**Issue by ABODFTW** [girder/django-s3-file-field#154]
README needs more details.

I'm not sure how to configure this to work, and what does S3FF_UPLOAD_STS_ARN do? and what value it should have?

Other details that should be added:
- The description in the advanced configurations is missing
- Security considerations
- API reference

If someone can point me in the correct direction I am willing to contribute.

Thanks

**Comment by brianhelba** [comment]
Agreed, the README is woefully out of date, and needs to be updated, particularly following the major refactor which happened recently.

Thanks for filing this issue to track progress. The details you brought up should all be addressed when the README is updated.

I should have time to work on this within the next ~2 weeks.

**Comment by ABODFTW** [comment]
Thanks 😊👍, let me know if I can help
